pkg/minikube/node/start.go

```diff
@@ -856,14 +857,32 @@ func tryRegistry(r command.Runner, driverName, imageRepository, ip string) {
 	}
 
 	opts = append(opts, fmt.Sprintf("https://%s/", imageRepository))
-	if rr, err := r.RunCmd(exec.Command("curl", opts...)); err != nil {
+	exe := "curl"
+	if runtime.GOOS == "windows" {
+		exe = "curl.exe"
+	}
+	cmd := exec.Command(exe, opts...)
+	if rr, err := r.RunCmd(cmd); err != nil {
+		outside := true
 		klog.Warningf("%s failed: %v", rr.Args, err)
-		out.WarningT("This {{.type}} is having trouble accessing https://{{.repository}}", out.V{"repository": imageRepository, "type": driver.MachineType(driverName)})
-		out.ErrT(style.Tip, "To pull new external images, you may need to configure a proxy: https://minikube.sigs.k8s.io/docs/reference/networking/proxy/")
+
 		// using QEMU with the user network
 		if driver.IsQEMU(driverName) && ip == "127.0.0.1" {
 			out.WarningT("Due to DNS issues your cluster may have problems starting and you may not be able to pull images\nMore details available at: https://minikube.sigs.k8s.io/docs/drivers/qemu/#known-issues")
 		}
+		// now we shall also try whether this registry is reachable outside the machine
```

Comment:
We should skip the second check if the user is using the `none` or `ssh` driver since there is no different between an "inside" and "outside" check on the `none` driver, and checking the `host` on the `ssh` driver is not helpful.